src/components/item/iterator.js

```diff
@@ -19,13 +19,12 @@ const {
 
 class ItemIterator extends Iterator {
 
-  constructor(props) {
-    super(props)
-    this.state = {
-      isDragging: false
-    }
+
+  state = {
```

Comment:
No, you need to remove this too: this is just syntactic sugar and will mess with the state set by the super constructor.

Reply:
Done